api/resources_portal/views/material_request.py

```diff
@@ -150,12 +152,15 @@ def update(self, request, *args, **kwargs):
 
         if request.user == material_request.requester:
             if "irb_attachment" in request.data:
-                material_request.irb_attachment = serializer.validated_data["irb_attachment"]
+                irb = serializer.validated_data["irb_attachment"]
+                material_request.irb_attachment = irb
+                # allow the assignee read access to the attachment
+                assign_perm("view_attachment", material_request.assigned_to, irb)
 
             if "requester_signed_mta_attachment" in request.data:
-                material_request.requester_signed_mta_attachment = serializer.validated_data[
-                    "requester_signed_mta_attachment"
-                ]
+                signed_mta = serializer.validated_data["requester_signed_mta_attachment"]
+                material_request.requester_signed_mta_attachment = signed_mta
+                assign_perm("view_attachment", material_request.assigned_to, signed_mta)
```

Comment:
Same answer to https://github.com/AlexsLemonade/resources-portal/pull/211/files#r442897370 applies here as well.